admin/c2cgeoportal_admin/routes.py

```diff
@@ -13,11 +13,12 @@ def includeme(config):
 
     config.add_route('home', '/')
 
-    from c2cgeoportal_commons.models.main import Role
+    from c2cgeoportal_commons.models.main import (Role, LayerWMS, LayerWMTS)
```

Comment:
What are the parenthesis purpose ?

Reply:
parenthesis are needed only in multiline statements